Exercises/Java Scrip/Ex_4/index.js

```diff
@@ -0,0 +1,5 @@
+alert("Welcome!");
+let firstNumber = prompt("Enter First Number");
+let secondNumber = prompt("Enter Second Number");
```

Comment:
As a good practice, before comparing numbers, make sure to get them converted to numbers. 

You are doing a string comparison here